Graylog2/graylog2-server
# Update some CEF fields to use BigInteger instead of Integer

Updates some CEF fields to use BigInteger instead of Integer.

## Description

Updates CEF Parser to use BigInteger for some fields to accommodate values larger than the max Integer value.

## Motivation and Context


The Max and Min Integer size is not being specified in the CEF specification and some users have logs values that exceed the maximum value able to be stored in a Java Integer. Because Graylog uses Java Integers to store many of these values, larger values were causing errors resulting in data was being lost

More details in issue: https://github.com/Graylog2/graylog2-server/issues/7371

## How Has This Been Tested?



Locally in development environment with CEF messages whose values exceed the maximum Java Integer value.

Steps to reproduce:
1. Start a CEF TCP input
2. Send log message to input e.g.:
 `echo -n -e '10.217.31.247 CEF:0|Citrix|NetScaler|NS10.0|APPFW|APPFW_STARTURL|6|src=10.217.253.78 spt=1234 method=GET request=http://vpx247.example.net/FFC/login.html msg=Disallow Illegal URL. cn1=1234 cn2=205 cs1=profile1 cs2=PPE0 cs3=AjSZM26h2M+xL809pON6C8joebUA000 cs4=ALERT cs5=2012 act=blocked in=8747876628 out=8747876628 sourceTranslatedPort=65536'"\0" | nc -w0 localhost 5555`
3. Verify error in logs

## Screenshots (if appropriate):

## Types of changes


## Checklist:
